### PR TITLE
Completed javascript-methods-Chris Mendoza

### PR DESCRIPTION
**Pull request by CTMendoza**
javascript-methods

### CODE REVIEW

File: javascript-methods/main.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+
+library.push(js);
+
+library.unshift(css);
+
+library.splice(1, 0);
```
Comment (BrettAlbright) [inline javascript-methods/main.js]: the second argument should be a 1!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiniest of updates

**Review by Zhiwutian — APPROVED** [review]
👍🏼

File: javascript-methods/main.js (hunk 1 above)
Comment (CTMendoza) [reply javascript-methods/main.js]: javascript-methods

Updated screenshot

**Pull request merged**